.github/workflows/prebuild-docker.yml

```diff
@@ -2,9 +2,10 @@ name: Prebuild Docker image
 
 on:
   push:
-    branches:
-    - main
-
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Do we want to rebuild the docker image on all pull requests too?

Reply:
It is an 8-minunte action that detects the breaking changes. I thought, why not.
Previously the break stayed unnoticed for 3 months.

Reply:
Added a new GitHub action triggered in the PR.